src/content/blog/astro-330.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: "Astro 3.3: <Picture /> component"
+description: "Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatibility for syntax highlighting, and package provenance."
+publishDate: "October 12, 2023"
+authors:
+  - nate
+  - erika
+  - bjorn
+  - ema
+coverImage: "/src/content/blog/_images/astro-330/blog-hero-3_3-release.webp"
+socialImage: "/src/content/blog/_images/astro-330/og-image-3_3-release.webp"
+lang: "en"
+---
+
+Astro 3.3 is out with a brand new, experimental `<Picture />` component, improved compatibility for syntax highlighting, package provenance, and a handful of other quality of life improvements.
+
+- **[New `<Picture />` component](#picture-component)**
+- **[Syntax Highlighting improvements](#syntax-highlighting)**
+- **[Package Provenance](#package-provenance)**
+- **[Quality of Life Improvements](#quality-of-life-improvements)**
+
+```
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Picture Component
+
+When we announced [Better Images in Astro](/blog/images/) with our built-in image optimization feature, we were proud to share our new vision for working with images. In 3.0 we removed our old image integration, and along with it some existing features we know you were using, to focus on building back better. We promised our image story was just getting started.
```

Comment:
```suggestion
When we announced [Better Images in Astro](/blog/images/) with our built-in image optimization feature, we were proud to share our new vision for working with images. In 3.0 we removed our old image integration, and along with it some existing features we know you were using. We decided to build back better, with a stronger image story that was only getting started.
```

You might change this entirely, and go in a different direction. But this at least makes the sentences there more manageable.  :smile:

Reply:
This is great!